package.json

```diff
@@ -45,15 +48,15 @@
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.7.5",
-    "@angular/cli": "^6.1.5",
+    "@angular-devkit/build-angular": "~0.10.3",
+    "@angular/cli": "^7.0.3",
```

Comment:
We could update the version in [README](https://github.com/thehyve/glowing-bear#glowing-bear) too

Reply:
updated :)